pkg/upstream/http.go

```diff
@@ -20,6 +21,10 @@ const (
 
 	httpScheme  = "http"
 	httpsScheme = "https"
+
+	defaultFlushInterval = 1 * time.Second
```

Comment:
I wonder if default constants for these are best included in `options/upstreams.go` adjacent to all the field descriptions? @JoelSpeed - I think this is a question for you to make sure we align everything to your vision for how V7 structured configs will map to options structs

Reply:
I had the same thought, I do see now that the `options` package is already included in this file, and is obviously available to the `legacy_options` as well, so moving these defaults to a single, sane place seems like the best option for everyone. I'll include this in my next commit. That is of course unless V7 has new/mo-betta opinions @JoelSpeed.